plugins/woocommerce-admin/client/marketplace/marketplace.scss

```diff
@@ -3,6 +3,11 @@
 .woocommerce-admin-page__extensions {
 	background: #fff;
 
+	#wpbody-content {
+		/* Prevent double-scrollbar issue on WooCommerce > Extension pages */
+		overflow: hidden;
```

Comment:
I was going to suggest `overflow-y` here, but I see `_reset.scss` already gives this `overflow-x: hidden !important;`.

Reply:
Yeah; it eventually turned out that this approach was the cleanest way to avoid _another_ `!important`.